src/soc/amd/rome/src/lib.rs

```diff
@@ -3,30 +3,32 @@
 
 pub mod hsmp;
 
+use hsmp::HSMP;
+
 pub fn soc_init(w: &mut impl core::fmt::Write) -> Result<(), &'static str> {
```

Comment:
I notice there is no error return. Should there be?

Reply:
Eventually yes--but right now, we are not using HSMP for anything. So it doesn't make sense to stop the entire boot process (for example) just because it returns something weird.